SpatialGDK/Source/SpatialGDKFunctionalTests/SpatialGDK/SpatialTestRepNotify/SpatialTestRepNotify.cpp

```diff
@@ -24,8 +29,11 @@
 ASpatialTestRepNotify::ASpatialTestRepNotify()
 	: Super()
 {
-	Author = "Miron + Andrei";
+	Author = "Miron + Andrei + Arthur";
 	Description = TEXT("Test RepNotify replication and shadow data");
+
+	TestSubobject = CreateDefaultSubobject<USpatialTestRepNotifySubobject>(TEXT("USpatialTestRepNotifySubobject"));
```

Comment:
Nit - I'm not sure if I feel that good about making this the Test's subobject as opposed to putting all of this onto a separate test actor - easier to do cleanup. This was present in the test before though, so 🤷

Reply:
I think you're right - I'll split it off